test-reminder-email.sh

```
@@ -0,0 +1,63 @@
+# 2. Test the POST endpoint with a complete request
+echo -e "\n${BLUE}2. Testing POST endpoint with a complete request...${NC}"
+
+curl -s -X POST "http://$HOST:$PORT/api/reminders/send" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "plantName": "Script Test Plant",
+    "reminderType": "Water",
+    "reminderDate": "'$(date -I)'",
+    "reminderTime": "10:00 AM",
+    "notes": "This is a test from the bash script",
+    "userEmail": "'$EMAIL'",
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)